plugins/inputs/logfile/logfile_test.go

```diff
@@ -434,7 +434,7 @@ func createWriteRead(t *testing.T, prefix string, logFile *LogFile, done chan bo
 	defer close(evts)
 	// Choose a large enough number of lines so that even high-spec hosts will not
 	// complete receiving logEvents before the 2nd createWriteRead() goroutine begins.
-	const numLines int = 1000000
+	const numLines int = 10000
```

Comment:
Changed this to resolve macOS build issue

Reply:
I think before, this was 100,000. Did you need to lower it down 2 orders of magnitude for the test to pass? @adam-mateen how long would it take to write an adequate integration test that validates the auto removal behavior?